src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml

```diff
@@ -755,13 +755,13 @@
    <text:p text:style-name="Text_20_body">The OpenDocument specification defines conformance for packages, package consumers, and package producers, with two conformance classes called conforming and extended conforming. This chapter defines the basic requirements for the individual conformance targets.</text:p>
    <text:h text:style-name="Heading_20_2" text:outline-level="2"><text:bookmark-start text:name="__RefHeading__752789_826425813"/>Package Conformance<text:bookmark-end text:name="__RefHeading__752789_826425813"/></text:h>
    <text:h text:style-name="Heading_20_3" text:outline-level="3"><text:bookmark-start text:name="__RefHeading__752791_826425813"/><text:reference-mark-start text:name="OpenDocument Package"/>OpenDocument Package<text:bookmark-end text:name="__RefHeading__752791_826425813"/><text:reference-mark-end text:name="OpenDocument Package"/></text:h>
-   <text:p text:style-name="Text_20_body">An <text:span text:style-name="Def">OpenDocument Package</text:span> <text:span text:style-name="ISO_20_Keyword">shall</text:span> meet the following requirements:</text:p>
```

Comment:
but here it's actually removed?

Reply:
Yes, fixed in the commit today, I overread that **every* keyword needs to be marked and assumed that no specific "*shall*" would be omitted. Fixed.